content/courses/program-optimization/program-architecture.md

```diff
@@ -755,21 +755,21 @@ pub fn run_concept_shared_account_redeem(ctx: Context<ConceptSharedAccountRedeem
 ```
 
 Here, in the `run_concept_shared_account` function, instead of transferring to
-the bottleneck, we transfer to the `donation_tally` PDA. This way, we’re only
+the bottleneck, we transfer to the `donation_tally` PDA. This way, we're only
 effecting the donator's account and their PDA - so no bottleneck! Additionally,
 we keep an internal tally of how many lamports need to be redeemed, ie be
 transferred from the PDA to the community wallet at a later time. At some point
 in the future, the community wallet will go around and clean up all the
 straggling lamports (probably a good job for
-[clockwork](https://www.clockwork.xyz/)). It’s important to note that anyone
+[clockwork](https://www.clockwork.xyz/)). It's important to note that anyone
```

Comment:
Clockwork is dead. Need a replacement for job scheduling. Does Pyth do this?

Also write it like "a job scheduling service like Whatever" - see CONTRIBUTING.md about not picking favorites.

Reply:
I didn't find any alternative after searching everywhere on the Internet. Pyth and Switchboard also do not support this. This line in the bracket is removed.  Let me know if you have an alternative suggestion on the same or feedback.